src/components/Account.js

```diff
@@ -1,63 +1,167 @@
 import React, { useState, useEffect } from 'react';
-import { useNavigate} from 'react-router-dom';
+
+import { Link } from 'react-router-dom';
 import axios from 'axios';
 import "./Account.css";
 
+const Account = () => {
 
-const Account = ({currentUser, setCurrentUser, setIsLoggedIn, token}) => {
   const [firstname, setFirstname] = useState("");
   const [lastname, setLastname] = useState("");
   const [email, setEmail] = useState("");
   const [username, setUsername] = useState("");
   const [address, setAddress] = useState("");
+  const [isAdmin, setIsAdmin] = useState("");
```

Comment:
We have this state both here and in App.js. It looks like we also have this information stored on the currentUser; is this necessary, or could we pass in currentUser to this component and call currentUser.isAdmin instead?